ExtractCopyright/Properties/AssemblyInfo.cs

```diff
@@ -8,11 +8,6 @@
 [assembly: AssemblyTitle("ExtractCopyright")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("SIL International")]
-[assembly: AssemblyProduct("ExtractCopyright")]
-[assembly: AssemblyCopyright("Copyright © 2017 SIL International")]
-[assembly: AssemblyTrademark("")]
-[assembly: AssemblyCulture("")]
```

Comment:
why is this information deleted?  is it incompatible with strong naming?  I would think that the company and copyright information would still be important information.